R/plotting.R

```diff
@@ -510,72 +620,98 @@ plotMotifHeatmaps <- function(x,
 
 #' @title Plot Stability Paths
 #'
-#' @description Plot the stability paths of each variable (predictor), 
+#' @description Plot the stability paths of each variable (predictor),
 #'   showing the selection probability as a function of the regularization step.
 #'
-#' @param se the \code{SummarizedExperiment} object resulting from stability 
+#' @param se the \code{SummarizedExperiment} object resulting from stability
 #'   selection, by running \code{\link[monaLisa]{randLassoStabSel}}.
 #' @param selProbMin A numerical scalar in [0,1]. Predictors with a selection
 #'   probability greater than \code{selProbMin} are shown as colored lines. The
 #'   color is defined by the \code{col} argument.
-#' @param col color of the selected predictors.
-#' @param lwd line width (default = 1).
-#' @param lty line type (default = 1).
-#' @param ylim limits for y-axis (default = c(0,1.1)).
-#' @param ... additional parameters to pass on to \code{matplot}.
-#'
-#' @return \code{TRUE} (invisibly).
-#' 
-#' @examples 
+#' @param col vector of colors specifying what to color selected and
+#'   non-selected predictors.
+#' @param linewidth line width.
+#' @param alpha line transparency of the stability paths.
+#' @param ylim limits for y-axis.
+#'
+#' @return a \code{ggplot2} object.
+#'
+#' @examples
 #' ## create data set
 #' Y <- rnorm(n = 500, mean = 2, sd = 1)
 #' X <- matrix(data = NA, nrow = length(Y), ncol = 50)
 #' for (i in seq_len(ncol(X))) {
 #'   X[ ,i] <- runif(n = 500, min = 0, max = 3)
 #' }
-#' s_cols <- sample(x = seq_len(ncol(X)), size = 10, 
+#' s_cols <- sample(x = seq_len(ncol(X)), size = 10,
 #'   replace = FALSE)
 #' for (i in seq_along(s_cols)) {
 #'   X[ ,s_cols[i]] <- X[ ,s_cols[i]] + Y
 #' }
-#'   
+#'
 #' ## reproducible randLassoStabSel() with 1 core
 #' set.seed(123)
 #' ss <- randLassoStabSel(x = X, y = Y)
 #' plotStabilityPaths(ss)
 #'
-#' @seealso \code{\link[stabs]{stabsel}} and \code{\link[graphics]{matplot}}
+#' @seealso \code{\link[stabs]{stabsel}}
 #'
 #' @importFrom SummarizedExperiment assay rowData colData
-#' @importFrom graphics matplot
+#' @import ggplot2
```

Comment:
Could we import specific functions from ggplot2 also here for consistency?

Reply:
Done